gui/pulsed/pulsed_maingui.py

```diff
@@ -3103,4 +3172,8 @@ def update_laser_data(self):
         self.lasertrace_image.setData(x=x_data, y=y_data)
         return
 
+    @QtCore.Slot()
+    def run_pg_benchmark(self):
 
+        self.pulsedmasterlogic().sequencegeneratorlogic().sigRunPgBenchmark.emit()
```

Comment:
This is an absolute no-no. Never emit signals from outside the QObject they belong to (see also comment in logic)